modules/gdscript/gdscript_parser.cpp

```diff
@@ -3102,15 +3102,15 @@ void GDScriptParser::_parse_block(BlockNode *p_block, bool p_static) {
 						Vector<Node *> args;
 						Vector<double> constants;
 
-						bool constant = false;
+						bool constant = true;
 
 						for (int i = 1; i < op->arguments.size(); i++) {
 							args.push_back(op->arguments[i]);
-							if (constant && op->arguments[i]->type == Node::TYPE_CONSTANT) {
+							if (op->arguments[i]->type == Node::TYPE_CONSTANT) {
+								if (!constant) break;
```

Comment:
You can just break the loop when setting `constant = false` since there's no chance it'll become `true` again.

Reply:
yeah better!